app/views/claims/pages/index.html.erb

```diff
@@ -1,26 +1,49 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">It works! 🎉</h1>
+<%= render(NavigationBar.new(
+    items: navigation_items(@provider),
+    current_path: request.path,
+    current_user: User.first
+  )) %>
+    <div class="govuk-grid-row">
+    <br><br>
+      <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Organisation details</h1>
+      <%= govuk_summary_list do |summary_list|
+        summary_list.with_row do |row|
+        row.with_key(text: "Organisation name")
+        row.with_value(text: @school.name)
+        end;summary_list.with_row do |row|
+          row.with_key(text: "UK provider reference number (UKPRN)")
+          row.with_value(text: @school.ukprn)
+        end;summary_list.with_row do |row|
+            row.with_key(text: "Unique reference number (URN)")
+            row.with_value(text: @school.urn)
+      end;end %>
 
-    <p class="govuk-body">
-      Your application is ready - so long as this page rendered without any errors you're good to go.
-    </p>
-
-    <%= govuk_summary_list(
-      rows: [
-        { key: { text: "Service" }, value: { text: "Claims" } },
-        { key: { text: "Rails version" }, value: { text:  Rails.version } },
-        { key: { text: "Ruby version" }, value: { text:  RUBY_VERSION } },
-        { key: {
-          text: "GOV.UK Frontend" },
-          value: {
-            text: JSON
-              .parse(File.read(Rails.root.join("package.json")))
-              .dig("dependencies", "govuk-frontend")
-              .tr("^", "")
-          }
-        }
-      ]
-    ) %>
-  </div>
-</div>
+        <h2 class="govuk-heading-m govuk-!-margin-top-9">Contact details</h2>
+        <%= govuk_summary_list do |summary_list|
+          summary_list.with_row do |row|
+          row.with_key(text: 'Email address')
+          row.with_value do %>
+            <a href="contact@example.com">contact@example.com</a>
+          <% end;row.with_action(text: 'Change', visually_hidden_text: 'email address', href: '/contact-details')
```

Comment:
Can we use translations for the text and visually hidden text? Same with the other actions